packaging/packager_rpm.sh

```diff
@@ -5,40 +5,44 @@ readonly distro="${DISTRO:-fedora}"
 readonly distro_version="${VERSION:-30}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# rpm dependencies
-# python3
-dnf -y install python3 python3-coverage
-# python-imaging
+echo "------------- python3 -------------"
+dnf -y install python3 python3-coverage python3-devel
```

Comment:
I assume we now need the `python3-devel` and didn't do so before for some reason